### PR TITLE
#875 time column showing in chart although it should not

### LINKED ISSUE #875: Chart Editor: Link Data to Simulation also adds "Time"
When I select a simulation output with the option "Link Data to Simulations", the "Time" repository is also added:

image

### PR DESCRIPTION
**Pull request by georgeDaskalakis**
closes #875

### CODE REVIEW

File: tests/MoBi.Tests/Presentation/SimulationChartPresenterSpecs.cs [hunk 1]
```diff
@@ -143,4 +137,25 @@ public void the_event_must_have_been_called_once()
          _observedDataAddedEventCounter.ShouldBeEqualTo(1);
       }
    }
+
+   public class When_adding_observed_data_to_the_char : concern_for_SimulationChartPresenter
```
Comment (Yuri05) [inline tests/MoBi.Tests/Presentation/SimulationChartPresenterSpecs.cs:141]: typo: char=>chart

Comment (georgeDaskalakis) [reply tests/MoBi.Tests/Presentation/SimulationChartPresenterSpecs.cs]: corrected

**Comment by Yuri05** [comment]
> @Yuri05 if everything else seems OK, should we merge?

I think so. I cannot judge if the change does not have any side effects, though.

**Comment by georgeDaskalakis** [comment]
> > @Yuri05 if everything else seems OK, should we merge?
> 
> I think so. I cannot judge if the change does not have any side effects, though.

Hmm OK, let's discuss briefly next week with @msevestre . Tbh, I cannot think of a scenario where we really need to explicitly show the time column.

**Pull request merged**